examples/enums/enums.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+
+// An enum is defined using a module enum declaration.
+enum Color {
+  RED,
+  GREEN,
+  BLUE
+}
+
+enum Language {
+    // An enum member can explicitly specify an associated
+    // expression.
+    ENG = "English",
+    TL = "Tamil",
+    SI = "Sinhala"
+}
+
+// An enum declaration can consist of a combination of members with or without
+// explicit associated values.
+enum Bands {
+    QUEEN,
+    PF = "Pink " + "Floyd"
+}
+
+public function main() {
+    // An enum member can be used in the same way as a string constant.
+    string skyColor = BLUE;
+    io:println("skyColor: ", skyColor);
```

Comment:
Shall we put a meaningful print statement there, "Color of Sky: " or something similar, not just the variable name.